core/lib/wallet_creator/src/main.rs

```diff
@@ -0,0 +1,28 @@
+use std::env;
+
+use rif_rollup_wallet_generator_lib::create_new_wallet;
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+    let args: Vec<String> = env::args().collect();
+
+    println!("{:?}", args);
+
+    let network: &str = &args[1];
```

Comment:
If I try to run the command without `zk` and without specifying the network param, I get an `index out of bounds` error. Could we please check if args length, before accessing them?

Reply:
Done